### ISSUE TITLE
Predefined External Dynamic Lists not included in security policies

### ISSUE THREAD

**Issue by ctktfg** [PaloAltoNetworks/iron-skillet#14]
Regardless of the "INCLUDE_PAN_EDL" setting, the PAN EDLs are not included in the security policies.

**Comment by scotchoaf** [comment]
can you confirm the sw version and what method/tools to set the value? Had found this error a few weeks ago and seeing if anything missed for a fix

**Comment by ctktfg** [comment]
I am running the version shown below:
```
git describe --tags
v1.0.3-137-g25a449c
```

To set the value, in tools/config_variables.yaml:
```
  - name: INCLUDE_PAN_EDL
    description: include the predefined Palo Alto Networks external lists
    value: True
```

**Comment by scotchoaf** [comment]
fixed for 8.1 and updating 8.0/9.0. Also now requires use of 'yes' as the value (with quotes). issue with reserved words and boolean vs text string issues

**Comment by scotchoaf** [comment]
an issue with mix of boolean and text not picking up the yes/no toggle. should be resolved for all skillets in 8.0, 8.1, 9.0
